xrpackage.js

```diff
@@ -1467,7 +1467,21 @@ export class XRPackageEngine extends XRNode {
       const inputPosition = localVector
         .copy(input.position)
         .divideScalar(this.scale);
-      const ps = this.children
+
+      const grabbables = [];
+      const getGrabbables = (p) => {
+        if (p.children.length > 0) {
+          p.recurseChildren(getGrabbables)
+        }
+        else if(!p.isAvatar) {
+          // console.log("Adding grabbable ", p.name)
+          grabbables.push(p)
+        }
+      }
+
+      this.recurseChildren(getGrabbables)
+
+      const ps = grabbables
         .sort((a, b) => {
```

Comment:
we could probably get rid of this sort if we checked for proximity in `getGrabbables` in the `else if` block.